packages/terra-avatar/src/common/Avatar.module.scss

```diff
@@ -0,0 +1,178 @@
+@import '~terra-mixins/lib/Mixins';
+
+:local {
+  .avatar {
+    align-items: center;
+    background-clip: padding-box; // Stops background color from bleeding through border on IE.
+    border: var(--terra-avatar-border, solid 1px #fff);
+    border-radius: 50%;
+    color: var(--terra-avatar-color, #fff);
+    display: inline-flex;
+    height: var(--terra-avatar-initial-height, 3.286em);
+    justify-content: center;
+    line-height: 0;
+    min-height: 1.143em;
+    min-width: 1.143em;
+    overflow: hidden;
+    position: relative;
+    vertical-align: middle; // Fixes gap below image variant
+    width: var(--terra-avatar-initial-width, 3.286em);
+  }
+
+  // Addresses https://github.com/cerner/terra-core/issues/1845
+  // Applies to images. Will be overriden by color variants.
+  .avatar::after {
+    border-radius: 100%;
+    box-shadow: var(--terra-avatar-after-box-shadow, inset 0 1px 3px 0 rgba(17, 22, 25, 0.3));
+    content: '';
+    display: block;
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+
+  .avatar > .icon {
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    display: block;
+
+    /* Disabling so that single quotes can be used in svg string */
+    /* stylelint-disable string-quotes */
+    &.facility {
+      background-image: var(--terra-avatar-icon-facility-background-image, inline-svg("<svg data-name='Layer_1' xmlns='http://www.w3.org/2000/svg' fill='#fff' viewBox='0 0 48 48'><path d='M33,20.2V12H15v8.2H9.8V48h12v-9h4.5v9h12V20.2H33z M17.2,42.8h-3V39h3V42.8z M17.2,33.8h-3V30h3V33.8z M25.5,33.8h-3V30h3 V33.8z M29.2,22.5h-3.8v3.8h-3v-3.8h-3.8v-3h3.8v-3.8h3v3.8h3.8V22.5z M33.8,42.8h-3V39h3V42.8z M33.8,33.8h-3V30h3V33.8z'/></svg>"));
+      height: var(--terra-avatar-icon-facility-height, 95%);
+      width: var(--terra-avatar-icon-facility-width, 95%);
+    }
+
+    &.shared-user {
+      background-image: var(--terra-avatar-icon-multi-user-background-image, inline-svg("<svg data-name='Layer_1' xmlns='http://www.w3.org/2000/svg' fill='#fff' viewBox='0 0 48 48'><path d='M15 0a8.7 8.7 0 1 1-8.7 8.7A8.71 8.71 0 0 1 15 0zm18 11a8.7 8.7 0 1 1-8.7 8.7A8.71 8.71 0 0 1 33 11zm15 37v-4.4a12.93 12.93 0 0 0-12.9-12.9h-4.2A12.93 12.93 0 0 0 18 43.6V48zM25.5 28.7a11.32 11.32 0 0 1-4.1-8.3 14.21 14.21 0 0 0-4.2-.7H13A13 13 0 0 0 0 32.6V37h16.4a15.52 15.52 0 0 1 9.1-8.3z'/></svg>"));
+      height: var(--terra-avatar-icon-multi-user-height, 70%);
```

Comment:
nit: I'd change `multi-user` to `shared-user` in the variable name to match how we are referring to this version of the component.

Reply:
updated https://github.com/cerner/terra-core/pull/2061/commits/d9eca3475f6642229e82b08f42a7627a1548c481